build-utils/css/constants/theme/day/colors/_blue.ts

```diff
@@ -1,22 +1,22 @@
 export const blue = {
   /** @info #DEEDFE  */
-  'blue-100': '#DEEDFE',
+  'blue-100': '#E7F6FE',
```

Comment:
Can you also update the comment jsdocs above to match with the value of the variable?

Reply:
Ah yes. Ill do that